requirements.txt

```diff
@@ -1,5 +1,5 @@
 av==10.*
-ctranslate2>=3.17,<4
+ctranslate2>=3.21,<4
```

Comment:
New release 3.22.0 on Ctranslate2 is in progress: https://github.com/OpenNMT/CTranslate2/pull/1561  
Please update this accordingly.

Reply:
will do, please ping when it's on pypi

Reply:
https://github.com/OpenNMT/CTranslate2/releases/tag/v3.22.0